packages/pyxis-react/src/commons/hooks/useDropdown.ts

```diff
@@ -0,0 +1,41 @@
+import {
+  Dispatch, MouseEvent, SetStateAction, useEffect, useState,
+} from 'react';
+import { Item } from 'internalComponents/Dropdown/items';
+
+export const initialDropdownConfig = { isOpen: false, activeDescendant: undefined };
+
+const useDropdown: UseDropdown = (items) => {
+  const [dropdown, setDropdown] = useState<DropdownConfig>(initialDropdownConfig);
+  const [mousePosition, setMousePosition] = useState<MousePosition | undefined>();
+  const handleMouseEnter: MouseEnterHandler = (value: string) => ({ screenX: x, screenY: y }) => {
+    if (mousePosition && x === mousePosition.x && y === mousePosition.y) return;
```

Comment:
I think you could write

```suggestion
    if (x === mousePosition?.x && y === mousePosition?.y) return;
```